### PR TITLE
Fix install interface location

### PR DESCRIPTION
**Pull request by rbsheth**
Hunter builds were failing because I put the `CMAKE_INSTALL_PREFIX` into the `INTERFACE_INCLUDE_DIRECTORIES`. While fixing that, I rebased to the latest master `b381467`

PTAL @ruslo

### CODE REVIEW

**Comment by ruslo** [comment]
Released: https://github.com/hunter-packages/assimp/releases/tag/v5.0.0-b381467-p0

File: code/CMakeLists.txt [hunk 1]
```diff
@@ -1119,24 +1195,37 @@ ENDIF(APPLE)
...
+    INCLUDE_DIRECTORIES("../")
+  endif (UNZIP_FOUND)
+ENDIF(NOT HUNTER_ENABLED)
 
 # Add RT-extension library for glTF importer with Open3DGC-compression.
 IF (RT_FOUND AND ASSIMP_IMPORTER_GLTF_USE_OPEN3DGC)
   TARGET_LINK_LIBRARIES(assimp ${RT_LIBRARY})
 ENDIF (RT_FOUND AND ASSIMP_IMPORTER_GLTF_USE_OPEN3DGC)
 
+IF(HUNTER_ENABLED)
```
Comment (ruslo) [inline code/CMakeLists.txt:1212]: Same as with C++ headers. I have to note that `EXPORT` is a general CMake feature, it's not some specific Hunter requirement. `EXPORT` can be used without Hunter too. `if(HUNTER_ENABLED)` is logically not correct.

Comment (rbsheth) [reply code/CMakeLists.txt:1212]: Yes, I did this explicitly to avoid touching their existing export/config logic.

**Comment by rbsheth** [comment]
@ruslo Upstream PR was merged to make updating in Hunter a bit easier: https://github.com/assimp/assimp/pull/2488

**Pull request merged**